www-src/app/replicator/main.coffee

```diff
@@ -523,26 +520,3 @@ module.exports = class Replicator extends Backbone.Model
     stopRealtime: =>
         @replicationLauncher?.stop()
         delete @replicationLauncher
-
-    # Update cache files with outdated revisions. Called while backup<
-    syncCache:  (callback) ->
```

Comment:
We still need this :
If wifi only option is checked, the replication (which is still activated) updates the File Meta data, but, cached binaries are not updated. 
We have to call this when wifi is back to maintain the synchronization of cached files.